Ginger/GingerCore/ALM/Qtest/ImportFromQtest.cs

```diff
@@ -0,0 +1,1128 @@
+                        }
+                        else
+                        {
+                            //try just to set the value
+                            try
+                            {
+                                stepActivityVar.Value = paramSelectedValue;
+                                if (stepActivityVar is VariableString)
+                                    ((VariableString)stepActivityVar).InitialStringValue = paramSelectedValue;
+                            }
+                            catch (Exception ex) { Reporter.ToLog(eLogLevel.ERROR, $"Method - {MethodBase.GetCurrentMethod().Name}, Error - {ex.Message}", ex); }
+                        }
+
+                        //add linked variable if needed
+                        if (string.IsNullOrEmpty(linkedVariable) == false)
+                        {
+                            stepActivityVar.LinkedVariableName = linkedVariable;
+                        }
+                        else
+                            stepActivityVar.LinkedVariableName = string.Empty;//clear old links
+                    }
+                }
+
+                //order the Activities Group activities according to the order of the matching steps in the TC
+                try
+                {
+                    foreach (QtestTestStep step in tc.Steps)
+                    {
+                        int stepIndx = tc.Steps.IndexOf(step) + 1;
+                        ActivityIdentifiers actIdent = (ActivityIdentifiers)tcActivsGroup.ActivitiesIdentifiers.Where(x => x.ActivityExternalID == step.StepID).FirstOrDefault();
+                        if (actIdent == null || actIdent.IdentifiedActivity == null) break;//something wrong- shouldn't be null
+                        Activity act = (Activity)actIdent.IdentifiedActivity;
+                        int groupActIndx = tcActivsGroup.ActivitiesIdentifiers.IndexOf(actIdent);
+                        int bfActIndx = busFlow.Activities.IndexOf(act);
+
+                        //set it in the correct place in the group
+                        int numOfSeenSteps = 0;
+                        int groupIndx = -1;
+                        foreach (ActivityIdentifiers ident in tcActivsGroup.ActivitiesIdentifiers)
+                        {
+                            groupIndx++;
+                            if (string.IsNullOrEmpty(ident.ActivityExternalID) ||
+                                    tc.Steps.Where(x => x.StepID == ident.ActivityExternalID).FirstOrDefault() == null)
+                                continue;//activity which not originally came from the TC
+                            numOfSeenSteps++;
+
+                            if (numOfSeenSteps >= stepIndx) break;
+                        }
+                        ActivityIdentifiers identOnPlace = (ActivityIdentifiers)tcActivsGroup.ActivitiesIdentifiers[groupIndx];
+                        if (identOnPlace.ActivityGuid != act.Guid)
+                        {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5395796)